packages/cli/test/functional/test_site_templates/test_default/expected/404.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="generator" content="MarkBind 4.0.2">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Page not found</title>
+  <link rel="stylesheet" href="markbind/css/bootstrap.min.css">
+  <link rel="stylesheet" href="markbind/fontawesome/css/all.min.css">
+  <link rel="stylesheet" href="markbind/glyphicons/css/bootstrap-glyphicons.min.css">
+  <link rel="stylesheet" href="markbind/css/octicons.css">
+  <link rel="stylesheet" href="markbind/material-icons/material-icons.css">
+  <link rel="stylesheet" href="markbind/css/codeblock-dark.min.css">
+  <link rel="stylesheet" href="markbind/css/markbind.min.css">
+  <script src="markbind/js/polyfill.min.js"></script>
+  <script src="markbind/js/vue.min.js"></script>
+  <script src="markbind/js/markbind.min.js"></script>
+  <script src="404.page-vue-render.js"></script>
+  <script src="markbind/js/jquery.min.js"></script>
+  <link rel="stylesheet" href="/plugins/markbind-plugin-anchors/markbind-plugin-anchors.css">
+  <link rel="stylesheet" href="/plugins/markbind-plugin-tree/markbind-plugin-tree.css">
+</head>
+<script>
+  const baseUrl = ''
+</script>
+
+<body>
+  <div id="app">
+    <p>
```

Comment:
I understand that this is generated code, but it looks a little weird in the placement of the `<p>` tag. Wonder if there's any underlying problem here, what do you think?

Reply:
I have no idea what exactly caused this but this occurred when the layout does not have any parent.

Right now ,layouts/404.md looks like this.
`{{ content }}`

When I added the div. The extra <p> tag is gone.
`<div>{{ content }}</div>`

Reply:
Could you paste the resultant html here? I think if it works fine we can include the `<div>` wrapping as u experimented

Reply:
@tlylt 
If you check the file, with the <div> added. The weird p tag is gone.